corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -1,11 +1,12 @@
 from django import template
 from django.utils import html
+from django.utils.html import format_html
 from django.utils.safestring import mark_safe
 from django.utils.translation import ugettext as _
 
 register = template.Library()
 
-EMPTY_LABEL = '<span class="label label-info">Empty</span>'
+EMPTY_LABEL = mark_safe('<span class="label label-info">Empty</span>')  # nosec: no user input
```

Comment:
i know you didn't add the original `Empty` text, but what about setting up global translations for the word "Empty"?

```
EMPTY_LABEL = format_html('<span class="label label-info">{}</span>', ugettext_lazy("Empty"))
```